lib/activerecord-import/import.rb

```diff
@@ -225,13 +225,31 @@ def support_setting_primary_key_of_imported_objects?
     #
     # ====  Using A Hash
     #
-    # The :on_duplicate_key_update option can be a hash of column name
+    # The :on_duplicate_key_update option can be a hash of column names
     # to model attribute name mappings. This gives you finer grained
     # control over what fields are updated with what attributes on your
     # model. Below is an example:
     #
     #   BlogPost.import columns, attributes, :on_duplicate_key_update=>{ :title => :title }
     #
+    # == On Duplicate Key Update (Postgres 9.5+)
+    #
+    # The :on_duplicate_key_update option is a hash with :key and
+    # :columns attributes. Below is an example:
+    #
+    #   BlogPost.import columns, values, :on_duplicate_key_update=>{ :key => :id, :columns => [ :date_modified, :content, :author ] }
```

Comment:
@zdennis I've added support for explicitly naming the conflicting constraint and I've added an example of that to the comments.